djnro/templates/front/management.html

```diff
@@ -17,12 +17,14 @@
     <h5>AAI Federation</h5>
     <hr>
     <p>{% blocktrans %}Authentication and authorization are carried out through a{% endblocktrans %} <a href="http://shibboleth.net/products/service-provider.html">Shibboleth SP</a>.</p>
-    <p>{% blocktrans %}The following attributes are required for administrators and must be released by their home IdPs to the SP{% endblocktrans %} {% trans "according to the" %} <a href="http://aai.grnet.gr/documentation">{% trans "policy and procedures documentation" %}</a> {% trans "provided by the" %} {{ FEDERATION_NAME }}:
+    <p>{% blocktrans %}The following attributes are required for administrators and must be released by their home IdPs to the SP{% endblocktrans %} {% trans "according to the" %} {% if FEDERATION_DOC_URL %}<a href="{{ FEDERATION_DOC_URL }} ">{% endif %}{% trans "policy and procedures documentation" %}{% if FEDERATION_DOC_URL %}</a>{% endif %} {% trans "provided by the" %} {{ FEDERATION_NAME }}:
```

Comment:
I tend to prefer using whole sentences in blocktrans blocks, moving out of the way any embedded markup/variables, as I believe that might lead to more concise translation.
But no need to change something for now.